src/main/java/com/rackspace/salus/resource_management/web/controller/ResourceApiController.java

```diff
@@ -186,4 +187,10 @@ public void delete(@PathVariable String tenantId,
   public Collection<String> getLabelNamespaces(@PathVariable String tenantId) {
     return resourceManagement.getLabelNamespaces();
   }
+
+  @GetMapping("/tenant/{tenantId}/search/{searchCriteria}")
+  @ApiOperation("Find all resources that match the searchCriteria in the ResourceId")
+  public PagedContent<Resource> getResourcesSearch(@PathVariable String tenantId, @PathVariable String searchCriteria, Pageable page) {
```

Comment:
```suggestion
  public PagedContent<Resource> getResourcesBySearchString(@PathVariable String tenantId, @PathVariable String searchCriteria, Pageable page) {
```

Maybe the same here too?